src/ocean/text/convert/Formatter.d

```diff
@@ -86,7 +86,7 @@ public istring format (Args...) (cstring fmt, Args args)
 
 *******************************************************************************/
 
-public mstring sformat (Args...) (ref mstring buffer, cstring fmt, Args args)
+public mstring aformat (Args...) (ref mstring buffer, cstring fmt, Args args)
```

Comment:
That's what I was thinking, if there is no conflict with just `format()` maybe we should use that.